kafka_utils/kafka_consumer_manager/commands/offset_manager.py

```diff
@@ -77,11 +79,15 @@ def preprocess_args(
                     groupid=groupid,
                 ),
             )
-        topics = cls.get_topics_from_consumer_group_id(
-            cluster_config,
-            groupid,
-            fail_on_error,
-        )
+        if storage == 'zookeeper':
+            topics = cls.get_topics_from_consumer_group_id(
+                cluster_config,
+                groupid,
+                fail_on_error,
+            )
+        else:
+            kafka_group_reader = KafkaGroupReader(cluster_config)
```

Comment:
I think get_topics_from_consumer_group_id should be able to fetch from either Kafka or Zookeeper. We should be able to move this logic inside that function.